### ISSUE TITLE
Make it possible for jack output to not connect to any port.

### ISSUE THREAD

**Issue by crocket** [MusicPlayerDaemon/MPD#550]
Jack output always tries to connect to some ports when mpd is started.
I have my own patchbay that dynamically disconnects mpd from ports and connect it to other ports.

It's good to prevent mpd from connecting to other jack ports.

**Comment by MaxKellermann** [comment]
If you leave `destination_ports` empty (or don't specify it at all), MPD will ask JACK for default ports to connect to. So you want MPD to not call `jack_connect()` at all?

**Comment by crocket** [comment]
That's correct. I want my jack patchbay to handle connections for every jack application.

**Comment by mr-berndt** [comment]
Seems this has not been resolved.

Yesterday I installed mpd. First try did nothing but I loaded a song. So on the second try it started directly with playback (also a dangerous default in my opinion) and it connected directly to my horn-driven tweeters, which require a crossover which brutefir is providing. All bypassend by mpd's directly connecting to system:playback_0/1 and I had a fullrange signal on a 113 dB / W tweeter! My luck and rescue was, that the music came with a fade in and I had time to jump an jerk the plugs out. Had id started with 0 dBfs that would have made over 135 dB for a short time and I guess I would be deaf for days as well as my tweeters would have been fried.

Pleas look into this, since in pro audio environments - where jack is often used - such behaviour can damage equipment as well as ears. In most jack environments people need full control over what's happening.

Thx! :)

**Comment by crocket** [comment]
I have moved onto pipewire.

**Comment by mr-berndt** [comment]
Yeah, I started with pipewire this time, coming from jack. At least on my target hw (intel atom / raspberrypi 3B), I find pipewire is far more dependent on potent hardware than jack, which in itself I don't read as a good sign. So I stay with jack.

**Comment by borine** [comment]
> Seems this has not been resolved

Are you sure about that? Have a look at the `auto_destination_ports` jack output plugin configuration option in the MPD user manual.

**Comment by mr-berndt** [comment]
That is there and that's what I am using currently. Still 'none' would be preferred, for many reasons.
